db/seeds.rb

```diff
@@ -94,26 +94,36 @@
     email: 'testytesterson@gmail.com',
     item_type: Item.gift_card,
     refunded: false,
-    amounts: [10_000, 8000]
+    amounts: [10_000, 8000],
+    single_use: false
   },
   {
     email: 'testytesterson2@gmail.com',
     item_type: Item.gift_card,
     refunded: true,
-    amounts: [7500, 3000]
+    amounts: [7500, 3000],
+    single_use: false
   },
   {
     email: 'testytesterson3@gmail.com',
     item_type: Item.gift_card,
     refunded: false,
-    amounts: [5000]
+    amounts: [5000],
+    single_use: false
+  },
+  {
+    email: 'testytesterson3@gmail.com',
```

Comment:
nit: should we make this testy testerson 4? 😜

Reply:
LOL yes

Reply:
😎